.github/CODEOWNERS

```
@@ -1 +1 @@
-* @AliceBalfanz @TonioF @dzelge @forman @pont-us
\ No newline at end of file
+* @AliceBalfanz @TonioF @dzelge @forman @pont-us @thomasstorm
```

```suggestion
* @AliceBalfanz @TonioF @forman @pont-us @thomasstorm
```

If we (i.e. some conda bots) change this file anyway, we can also remove Helge.

---

I updated the recipe maintainer list in `meta.yaml` already, and I *think* that that should be enough for conda-forge to update the code owners accordingly when the PR is merged. I'll check this after merge, and update directly in the main branch if it doesn't happen automatically.